integration/discovery/discovery_test.go

```diff
@@ -400,7 +400,7 @@ var _ = Describe("DiscoveryService", func() {
 		Expect(sess.Err).To(gbytes.Say(`failed constructing descriptor for chaincodes:<name:"mycc-lifecycle"`))
 
 		By("deploying chaincode using org1 and org2")
-		chaincodePath := components.Build("github.com/hyperledger/fabric/integration/chaincode/module")
+		chaincodePath := components.Build(nwo.ModuleCCPath())
```

Comment:
This completely hides what is getting built; not a good thing.

Reply:
It seemed like a common reference to pull out.